_el/exercises.md

```diff
@@ -1,169 +1,95 @@
 ---
 layout: default
 permalink: "/exercises/"
-title: "Exercise Types"
-objectives:
-- Describe four types of formative assessment exercises for
-  programming classes.
-- Describe two kinds of feedback on programming exercises that can
-  be given by automated tools.
+title: "Τύποι Ασκήσεων"
+στόχοι:
```

Comment:
Στα υπόλοιπα κεφάλαια υπήρχε το "Learning objectives". Ίσως για λόγους ομοιομορφίας να κρατήσουμε το "Μαθησιακοί στόχοι". Στην καινούργια αγγλική έκδοση βλέπω το "After reading this chapter, you will be able to...". Αν θελήσουμε να το αντικαταστήσουμε, προτείνω το "Μετά από τη μελέτη αυτού του κεφαλαίου, θα είστε σε θέση να..."

Reply:
Thanks @pdelias ! Fixed in a new commit.